cpp/include/cudf/functions.h

```diff
@@ -438,13 +438,18 @@ gdf_error gdf_prefixsum_i64(gdf_column *inp, gdf_column *out, int inclusive);
  * @Param num_cols The number of columns in the input set
  * @Param input The list of columns whose rows will be hashed
  * @Param hash The hash function to use
+ * @Param initial_hash_values Initial hash values to combine with column hash values
```

Comment:
Appreciate it if you could update documentation here as well: https://github.com/rapidsai/cudf/blob/branch-0.6/cpp/src/examples/documentation_example.cpp

Reply:
Done